org.aardappel.TreeSheets.appdata.xml

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+  <id>org.aardappel.TreeSheets.desktop</id>
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>Zlib</project_license>
+  <name>Treesheets</name>
+  <summary>A hierarhical spreadsheet / outliner productivity tool.</summary>
+  <summary xml:lang="de">Ein hierarchisches Notizbuch</summary>
+  <description>
+    <p>
+        Suitable for any kind of data organization, such as Todo lists, calendars,
+        project management, brainstorming, organizing ideas, planning, requirements
+        gathering, presentation of information, etc.
+    </p>
+    <p xml:lang="de">
+        Geeignet für jede Form von Datenorganisation, wie TODO-Listen, Kalender,
+        Projektmanagement, Brainstorming, Ideenplanung- und Organisierung, Anforderungserhebung,
+        Informationspräsentation, usw.
+    </p>
+    <p>
+        It shares many characteristics with more familiar tools, and augments their
+        abilities - like a spreadsheet, but supporting hierarchical data; like a mind
+        mapper, but more organized and compact; like an outliner, but in more than one
+        dimension; and like a text editor, but with structure.
+    </p>
+    <p xml:lang="de">
+        Das Programm teilt sich mehrere Charakteristiken mit bekannteren Werkzeugen und erweitert diese.
+        Wie eine Tabellenkalkulation, jedoch mit hierarchischen Feldern. Wie ein Texteditor, jedoch mit Struktur.
+    </p>
+  </description>
+  <screenshots>
+    <screenshot type="default">
+      <image>http://strlen.com/treesheets/docs/images/screenshots/screenshot_todo_half.png</image>
```

Comment:
Isn't there any Linux screenshot available?

Reply:
You are right, I completely missed that. I replaced the screenshot with a linux one.